source/manual/seeing-things-in-the-aws-console.html.md

```diff
@@ -0,0 +1,63 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Seeing things in the AWS Console
+section: AWS
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-06-01
+review_in: 3 months
+---
+
+To follow this guide, you'll need a `gds-users` AWS account. Follow
+[the User Management in AWS guide](source/manual/user-management-in-aws.html.md)
+to get one.
+
+## Seeing things in the AWS Console
+
+The AWS Console can seem impenetrable, with lots of options, some of
+them restricted by your role access level.
+
+### Logging in to the central `gds-users` account
+
+To log into the central `gds-users` account, [go to the AWS Console](https://console.aws.amazon.com/).
```

Comment:
You could save the `Enter gds-users` step below by updating this link to `https://gds-users.signin.aws.amazon.com/console`.